mono/arch/s390x/s390x-codegen.h

```diff
@@ -268,7 +268,7 @@ typedef struct {
 	char	xx : 4;
 	char	r3 : 4;
 	char	r4 : 4;
-} __attribute__ ((packed)) RRD_Format;
+} __attribute__ ((__packed__)) RRD_Format;
```

Comment:
But what is the reason for this change? It's GCC-specific syntax anyway, so why make it look even uglier?